numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -149,11 +149,17 @@ gentype_free(PyObject *v)
 
 
 static PyObject *
-gentype_power(PyObject *m1, PyObject *m2, PyObject *NPY_UNUSED(m3))
+gentype_power(PyObject *m1, PyObject *m2, PyObject *modulo)
 {
```

Comment:
It's not entirely clear why this function exists - it seems to re-implement the op/rop dispatch that `PyNumber_Power` handles?